x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview.tsx

```diff
@@ -104,7 +104,7 @@ export const EngineOverview: ReactFC<> = () => {
               </h2>
             </EuiTitle>
           </EuiPageContentHeader>
-          <EuiPageContentBody>
+          <EuiPageContentBody data-test-subj="appSearchEngines">
```

Comment:
Kibana recommends using `data-test-subj` for finding elements in tests and even has a `testSubjects` [service](https://www.elastic.co/guide/en/kibana/current/development-functional-tests.html#_kibana_services) helper that makes it easier to find elements by this attribute.